### ISSUE TITLE
Download fails with status 401: refreshToken is required.

### ISSUE THREAD

**Issue by auto-b64d** [Darkness4/withny-dl#13]
### Environment
- OS: Windows 11
- withny-dl version: 1.2.1

### Configurations
(Comments do not exist on actual files)
```yaml
# config.yaml
credentialsFile: 'credential.yaml'
```
```yaml
# credential.yaml
token: "someToken"
refreshToken: "someRefreshToken"
```

### Actual output
<details>
<summary>Output (without timestamp)</summary>

```
$ withny-dl watch -c config.yaml
INF github.com/Darkness4/withny-dl/cmd/watch/watch_command.go:149 > listening listenAddress=:3000
INF github.com/Darkness4/withny-dl/cmd/watch/watch_config.go:73 > initial config detected
INF github.com/Darkness4/withny-dl/cmd/watch/watch_config.go:190 > loaded new config
INF github.com/Darkness4/withny-dl/cmd/watch/watch_command.go:210 > no notifier configured
dummy notify:
title: config reloaded
message:
INF github.com/Darkness4/withny-dl/withny/api/client.go:294 > refreshing token
ERR github.com/Darkness4/withny-dl/withny/api/client.go:331 > unexpected status code error="unexpected status code: 401" method=POST response="{\"message\":\"refreshToken is required.\",\"status\":401}" status=401 url=https://www.withny.fun/api/auth/token
ERR github.com/Darkness4/withny-dl/withny/api/client.go:148 > failed to login error="unexpected status code: 401"
ERR github.com/Darkness4/withny-dl/withny/api/client.go:533 > failed to login to withny error="unexpected status code: 401"
FTL github.com/Darkness4/withny-dl/cmd/watch/watch_command.go:192 > failed to login error="unexpected status code: 401"
```
</details>

**Comment by Darkness4** [comment]
Fixed in this commit https://github.com/Darkness4/withny-dl/commit/3ce25f7e078a9b293103a1376a33cf0e936b5269.

I've also added some instructions on the token-based authentication: https://github.com/Darkness4/withny-dl?tab=readme-ov-file#token-based-authentication

TL;DR: Please note that the refresh token can only be used once, and you may need to fetch a new one. But don't worry! Withny-dl will automatically rotate and store the new refresh token in the encrypted file `/tmp/withny-dl.json`. The only time you'll need to retrieve a new one is when both `/tmp/withny-dl.json` and `credentials.yaml` use an invalid refresh token.
